tune_sklearn/tune_search.py

```diff
@@ -293,10 +474,18 @@ def _tune_run(self, config, resources_per_trial):
                 `tune.run`.
 
         """
+        stop_condition = {"training_iteration": self.max_iters}
         if self.early_stopping is not None:
             config["estimator"] = [
                 clone(self.estimator) for _ in range(self.n_splits)
             ]
+            if hasattr(self.early_stopping, '_max_t_attr'):
+                # we want to delegate stopping to schedulers which support it, but we want it to stop eventually, just in case
+                # the solution is to make the stop condition very big
+                stop_condition = {
+                    "training_iteration": self.max_iters * self.num_samples *
+                    1000
+                }
```

Comment:
Can you leave this as a TODO? `_max_t_attr` is not public API, but I think it's safer to leave as-is for now.